### PR TITLE
Config Feature Updates

### PR DESCRIPTION
**Pull request by mnkolev**
As a result of the new AWS config features, I have added the following classes as AWS Objects and respective properties within `config.py`. Please let me know if there are any changes that I need to make or anything I can do to assist!

- [AWS::Config::AggregationAuthorization](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-config-aggregationauthorization.html)
- [AWS::Config::ConfigurationAggregator](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-config-configurationaggregator.html)
            - AccountAggregationSources
            - OrganizationAggregationSource

### CODE REVIEW

File: troposphere/config.py [hunk 1]
```diff
@@ -89,6 +96,30 @@ class ConfigSnapshotDeliveryProperties(AWSProperty):
...
+class AccountAggregationSources(AWSProperty):
+    props = {
+        'AllAwsRegions': (boolean, False),
+        'AwsRegions': ([basestring], False),
+        'AccountIds': ([basestring], True),
+
+    }    
+
+class ConfigurationAggregator(AWSObject):
+    resource_type = "ConfigurationAggregator"
```
Comment (markpeek) [inline troposphere/config.py]: resource_type should be "AWS::Config::ConfigurationAggregator"

File: troposphere/config.py [hunk 2]
```diff
@@ -89,6 +96,30 @@ class ConfigSnapshotDeliveryProperties(AWSProperty):
...
+
+    props = {
+        'ConfigurationAggregatorName': (basestring, True),
```
Comment (markpeek) [inline troposphere/config.py]: Alphabetize

**Comment by mnkolev** [comment]
Apologies for missing those. Please let me know if I missed anything else and will get it sorted out

**Comment by markpeek** [comment]
Thanks for this change. I updated it to fix a couple of test failures and pushed it to master.

**Pull request closed without merging**